sql/codegen.go

```diff
@@ -117,6 +117,13 @@ func newFiller(pr *extendedSelect, ds *trainAndValDataset, fts fieldTypes, db *D
 	if err != nil {
 		return nil, err
 	}
+	if err == nil && r.Driver == "hive" {
```

Comment:
Do not need `err == nil` because the above IF statement does that.

Reply:
Done